velox/connectors/hive/storage_adapters/abfs/AbfsFileSystem.cpp

```diff
@@ -35,11 +35,20 @@ class AbfsReadFile::Impl {
 
  public:
   explicit Impl(std::string_view path, const config::ConfigBase& config) {
-    auto account = AbfsConfig(path, config);
-    filePath_ = account.filePath();
-    fileClient_ =
-        std::make_unique<BlobClient>(BlobClient::CreateFromConnectionString(
-            account.connectionString(), account.fileSystem(), filePath_));
+    auto abfsConfig = AbfsConfig(path, config, false);
+    filePath_ = abfsConfig.filePath();
+    if (abfsConfig.authType() == "SAS") {
```

Comment:
Let's add a function `getReadFileClient()` to AbfsConfig class move all this to that class.

Reply:
Updated.